examples/play_long_file.py

```diff
@@ -72,9 +72,9 @@ def callback(outdata, frames, time, status):
     assert not status
     try:
         data = q.get_nowait()
-    except queue.Empty:
+    except queue.Empty as e:
         print('Buffer is empty: increase buffersize?', file=sys.stderr)
-        raise sd.CallbackAbort
+        raise sd.CallbackAbort from e
```

Comment:
This will never bubble up to the user, see:

https://github.com/spatialaudio/python-sounddevice/blob/afe0a822b5591c2e3ae7fe9f1517d7342e048c46/sounddevice.py#L2648-L2651

But I guess it still doesn't hurt to add the `from e`, right?

Reply:
Yes, I think in this case it's better to stay correct even when there aren't any practical implications.